Test failing;
```
======================================================================
ERROR: custom.nic_compliance.tests.test_utils:TestDecodePassword.test_login_attempt
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/commcare-hq-ro/custom/nic_compliance/tests/test_utils.py", line 49, in test_login_attempt
    response = client.post(reverse('login'), form_data, follow=True)
  File "/vendor/lib/python2.7/site-packages/django/test/client.py", line 548, in post
    secure=secure, **extra)
  File "/vendor/lib/python2.7/site-packages/django/test/client.py", line 350, in post
    secure=secure, **extra)
  File "/vendor/lib/python2.7/site-packages/django/test/client.py", line 416, in generic
    return self.request(**r)
  File "/vendor/lib/python2.7/site-packages/django/test/client.py", line 501, in request
    six.reraise(*exc_info)
  File "/vendor/lib/python2.7/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/vendor/lib/python2.7/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/vendor/lib/python2.7/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/mnt/commcare-hq-ro/corehq/apps/domain/decorators.py", line 265, in wrapped_view
    return view_func(*args, **kwargs)
  File "/vendor/lib/python2.7/site-packages/django/views/decorators/debug.py", line 76, in sensitive_post_parameters_wrapper
    return view(request, *args, **kwargs)
  File "/mnt/commcare-hq-ro/corehq/apps/hqwebapp/views.py", line 388, in login
    return _login(req, domain, "login_and_password/login.html")
  File "/vendor/lib/python2.7/site-packages/django/views/decorators/debug.py", line 76, in sensitive_post_parameters_wrapper
    return view(request, *args, **kwargs)
  File "/mnt/commcare-hq-ro/corehq/apps/hqwebapp/views.py", line 335, in _login
    old_lang = req.session[LANGUAGE_SESSION_KEY]
  File "/vendor/lib/python2.7/site-packages/django/contrib/sessions/backends/base.py", line 57, in __getitem__
    return self._session[key]
KeyError: u'_language'
```

--- a/corehq/apps/hqwebapp/views.py
+++ b/corehq/apps/hqwebapp/views.py
@@ -332,7 +332,7 @@ def _login(req, domain_name, template_name):
 
     if 'auth-username' in req.POST:
         new_lang = CouchUser.get_by_username(req.POST['auth-username']).language
-        old_lang = req.session[LANGUAGE_SESSION_KEY]
+        old_lang = req.session.get(LANGUAGE_SESSION_KEY)
         update_session_language(req, old_lang, new_lang)
     if req.user.is_authenticated and req.method == "GET":
         redirect_to = req.GET.get('next', '')

